packages/config/src/projects/intmax/intmax.ts

```diff
@@ -21,14 +36,160 @@ export const intmax: ScalingProject = underReviewL2({
       ],
       repositories: ['https://github.com/InternetMaximalism'],
       socialMedia: ['https://twitter.com/intmaxIO'],
+      bridges: ['https://app.intmax.io/bridge'],
```

Comment:
They don't seem to have a block explorer for the mainnet yet, but maybe it would still make sense to link testnet explorer https://beta.testnet.explorer.intmax.io? But I don't have a strong opinion here, it is just for a testnet and could confuse someone

Reply:
yep i think it's confusing